### DISCUSSION TITLE
降低人力成本，友達數位 COMPUTEX 展出人機協作、AI 監控系統

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#639] [General]
https://technews.tw/2022/05/23/auo-computex-taipei-2022/
友達攜手子公司友達數位將在台北國際電腦展（COMPUTEX）首次展出 Class100 無塵室清潔機器人、智働巡檢模組及智慧燈塔 CPS 管理系統（Cyber-Physical Systems），透過人機協作及 AI 即時監控系統，有效降低工廠人力成本、改善作業環境品質及提升生產良率。

因應全球缺工浪潮，友達數位推出台灣首創 Class100 無塵室清潔機器人，具專利黏塵清潔技術，搭配智慧清潔路徑規劃，實現不發塵、不擾流之無塵室場域清潔任務。亦可擴充搭載氣體粒子計數器、溫溼度感測器等環境感測器，針對於無塵室環境進行即時監控，節省無塵室所需之清潔與管理人力，提升環境品質及產品良率。

**Comment by awkward-willy** [comment]
今年的 COMPUTEX 感覺有展出許多新奇的內容
尤其是台灣有相當完整的資訊科技產業鏈
有很多我們平常不太可能接觸到的產業與技術
未來有機會很想去一探究竟

**Comment by 11timyyy** [comment]
無塵室真的需要時時刻刻保持著乾淨、整潔

**Comment by jason79461385** [comment]
真的蠻想去computex 但是都沒有機會⋯
